kbase-extension/static/kbase/js/widgets/appWidgets2/input/newObjectInput.js

```diff
@@ -24,10 +24,12 @@ define([
                 value: undefined,
             },
             runtime = Runtime.make(),
-            otherOutputParams = config.closeParameters.filter((value) => {
-                return value !== spec.id;
-            });
-        let parent, container, dom;
+            otherOutputParams = config.closeParameters
```

Comment:
init this to an empty array if there are no params supplied so that later code does not fail